src/webapp/components/survey/SurveyForm.tsx

```diff
@@ -79,9 +79,27 @@ export const SurveyForm: React.FC<SurveyFormProps> = props => {
         }
     };
 
-    const updateQuestion = (question: Question) => {
+    const updateQuestion = (question: Question, stageId?: string) => {
```

Comment:
This kind of functions: saveSurveyForm, updateQuestion, addProgramStage, removeProgramStage are logic and I think they should be placed in a hook